sdk/servicebus/Azure.Messaging.ServiceBus/README.md

```diff
@@ -182,7 +182,7 @@ string body = receivedMessage.Body.ToString();
 Console.WriteLine(body);
 ```
 
-### Send and receive a batch of messages
+### Send a batch of messages
```

Comment:
We'll need to reflect the new sections in the list on L145, replacing the existing item.

Reply:
Hi @jsquire thank you for reviewing the PR.  The background for this change; a couple of engineers on the customer side mentioned the heading said **"Send and receive a batch of messages"** but there was no actual code for batch receiving of messages in the markdown file.

What I have done is go back to the original markdown file and just include the missing code which is supposed to receive a batch of messages.